src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -1027,7 +1027,7 @@ bool SequentialSfMReconstructionEngine::Resection(const size_t viewIndex)
     }
     if(!sfm::SfM_Localizer::RefinePose(
```

Comment:
I agree. Good catch, it will help to make things more explicit ;-).
The cons, is that perhaps more user will play with hidden parameters since they are more explicit :-(
Thx, I will update that soon.